lib/omnibus/s3_license_cache.rb

```diff
@@ -116,11 +131,10 @@ def populate
       # @param [String] destination
       #
       def get_object(software, license_file, destination)
-        object = client.bucket(Config.s3_bucket).object(key_for(software, license_file))
+        object = bucket.object(key_for(software, license_file))
+
         object.get(
-          response_target: destination,
-          bucket: Config.s3_bucket,
-          key: key_for(software, license_file)
+          response_target: destination
```

Comment:
From the [AWS SDK docs](https://docs.aws.amazon.com/sdk-for-ruby/v2/api/Aws/S3/Object.html#get-instance_method): the two parameters I removed here didn't do anything. The bbucket and key are already part of the `object`.